src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecPlanGraph/ActualPlanDataReaderNodeBuilder.cs

```diff
@@ -3,7 +3,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
-namespace Microsoft.SqlTools.ServiceLayer.ShowPlan.ShowPlanGraph
+namespace Microsoft.SqlTools.ServiceLayer.ExecutionPlan.ExecPlanGraph
```

Comment:
We should avoid shortening names unless necessary - `ExecutionPlanGraph` would be clearer here

Reply:
Using `ExectionPlanGraph` results in a build error because there's a class named `ExecutionPlanGraph` under the `/ExecutionPlan/Contracts/` directory that conflicts with it. Would renaming to something like `ExecutionGraph`, `PlanGraph`, or simply `Graph` be better since `ExecutionPlan` is already a part of the namespace?

Reply:
The namespace would then be this: `Microsoft.SqlTools.ServiceLayer.ExecutionPlan.Graph`